### ISSUE TITLE
Face detections in db don't respond to creation/removal of .nomedia

### ISSUE THREAD

**Issue by hashworks** [nextcloud/recognize#650]
### Which version of recognize are you using?

3.3.6

### Enabled Modes

Face recognition

### TensorFlow mode

Normal mode

### Which Nextcloud version do you have installed?

25.0.3

### Which Operating system do you have installed?

Arch Linux

### Which Docker container are you using to run Nextcloud? (if applicable)

None

### How much RAM does your server have?

64GiB

### What processor Architecture does your CPU have?

x86_64

### Describe the Bug

Some face albums show no picture:

![image](https://user-images.githubusercontent.com/2362091/214034575-0b91dde7-7e99-4d8c-8a14-b7243f088760.png)

While they say that they contain a bunch of photos, opening the album shows an empty page:
![image](https://user-images.githubusercontent.com/2362091/214034702-9f3596f5-3283-43e6-ac84-0ef7183cfcf9.png)

This shows no errors in the NC log.

### Expected Behavior

All albums contain working pictures.

### To Reproduce

Nothing special.

### Debug log

TODO

**Comment by LuPro** [comment]
I've encountered this issue myself and figured out it was because they were classified before I added a .nomedia file in a folder which caused the Photos App to not display the images but the "People" section still show the already classified (but now excluded) categories.

That or they got added to the queue in spite of the .nomedia, because I added that very early in the queuing process so it must have been very (un)lucky to get a few dozen images right at the start that have been excluded out of many tens of thousands that aren't.

**Comment by johnnyborg** [comment]
@hashworks 
can you check your network console of the browser (F12, tab network)? There a request should be an error there, what is it?

Had a similar issue, it recognized some old images in the trash (which i pruned). 
After that i ran qeury for it to fix it. Might still have it, need to check that tho.

**Comment by hashworks** [comment]
> can you check your network console of the browser (F12, tab network)? There a request should be an error there, what is it?

No error.

But according to the XML output the missing images are indeed in a folder where I added a `.nomedia` or `.noimage` file. Not sure what's a good solution here. Should recognize clean those up on a regular basis? Or is it up to the administrator to trigger a rescan?

There isn't really a cleanup command, one would need to trigger `recognize:reset-faces` and `recognize:classify`, right?

**Comment by marcelklehr** [comment]
Mh, yeah, that's non-trivial. I guess a listener for adding or removing ignore files would work.

**Comment by marcelklehr** [comment]
this is fixed in v3.6.0

**Comment by marcelklehr** [comment]
To reap the fruits of the fix in v3.6.0 you can simply remove and readd any .noimage or .nomedia files that haven't been followed.

**Comment by fbarbe00** [comment]
Hi, I installed recognize two days ago from the Apps (I'm running nextcloud 28.0.2), and I'm facing the same or a very similar issue. After a full scan, I added a .nomedia file in a bunch of directories that had already been scanned. I then triggered a full rescan (without deleting previous tags/faces). I now still see these images I don't want to see in the tags preview and people preview (and are included in the count). When I click on a specific tag/person, however, the pictures shown are only the ones that should be shown (not the .nomedia).
